### ISSUE TITLE
python 3.4 artifact build is broken (due to c-ares?)

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#10601]
https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/3204/architecture=x86,language=python,platform=linux/console

https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/3204/architecture=x64,language=python,platform=linux/console

```
warning: no files found matching '*.h' under directory 'src/python/grpcio/grpc'
warning: no files found matching '*.python' under directory 'src/python/grpcio/grpc'
warning: no previously-included files matching '*.so' found under directory 'src/python/grpcio/grpc/_cython'
warning: no previously-included files matching '*.pyd' found under directory 'src/python/grpcio/grpc/_cython'
error: [Errno 2] No such file or directory: 'third_party/c-ares'
+ rm -rf /tmp/pygrpc.qW1224
+ FAILED=true
+ '[' artifacts '!=' '' ']'
+ docker cp build_and_run_docker_e28b54c7-e2cf-4d2a-80b1-e85d496bea24:/var/local/git/grpc/artifacts /home/jenkins/workspace/gRPC_build_artifacts/architecture/x86/language/python/platform/linux
+ docker rm -f build_and_run_docker_e28b54c7-e2cf-4d2a-80b1-e85d496bea24
build_and_run_docker_e28b54c7-e2cf-4d2a-80b1-e85d496bea24
+ '[' true '!=' '' ']'
+ exit 1

2017-04-12 06:53:27,783 FAILED: build_artifact.python_linux_x86_cp34-cp34m [ret=1, pid=20137]
```

**Comment by y-zeng** [comment]
#10683 closed this issue automatically. But it still needs to be verified.

**Comment by y-zeng** [comment]
Python linux artifact builds have been fixed by #10683. But the windows artifact build is still broken.

**Comment by y-zeng** [comment]
#10781 fixed the windows artifact build by disabling c-ares for python on x64 windows.
https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/3271/
